thrift/agent.thrift

```diff
@@ -18,7 +18,7 @@ include "zipkincore.thrift"
 namespace cpp jaegertracing.agent.thrift
 namespace java io.jaegertracing.agent.thrift
 namespace php Jaeger.Thrift.Agent
-namespace netcore Jaeger.Thrift.Agent
+namespace netstd Jaeger.Thrift.Agent
```

Comment:
@Falco20019 could you elaborate on this change? Is it possible that both namespaces can remain?

Reply:
`netcore` is not available in Thrift 0.13 anymore. It was completely replaced with `netstd` which is compatible with .NET Core and .NET Framework (to some extend).

Reply:
I added both. In my tests I saw, that depending on 0.9.2 or 0.13.0 you will get the warning for either of them. 0.9.2 does not know `netstd`...